src/generators/java/mekanism/generators/client/gui/GuiIndustrialTurbine.java

```diff
@@ -100,9 +101,9 @@ public double getLevel() {
             TurbineMultiblockData multiblock = tile.getMultiblock();
             if (multiblock.isFormed()) {
                 storing = EnergyDisplay.of(multiblock.energyContainer);
-                producing = EnergyDisplay.of(MekanismConfig.general.maxEnergyPerSteam.get().divide(TurbineValidator.MAX_BLADES)
-                      .multiply(multiblock.clientFlow * Math.min(multiblock.blades,
-                            multiblock.coils * MekanismGeneratorsConfig.generators.turbineBladesPerCoil.get())));
+                producing = EnergyDisplay.of((long) ((MekanismConfig.general.maxEnergyPerSteam.get() / (double) TurbineValidator.MAX_BLADES)
+                                                     * (multiblock.clientFlow * Math.min(multiblock.blades,
+                      multiblock.coils * MekanismGeneratorsConfig.generators.turbineBladesPerCoil.get()))));
```

Comment:
Logically this seems correct, but formatting wise the spacing on this is rather screwed up. (Side note is there a chance of this overflowing? Do we need to use `MathUtils#clampToLong`?